launch/launch/actions/execute_local.py

```diff
@@ -182,7 +184,7 @@ def __init__(
             Use get_stdout() and get_stderr() to read the buffered output.
         :param: on_exit list of actions to execute upon process exit.
         :param: respawn if 'True', relaunch the process that abnormally died.
-            Defaults to 'False'.
+            Defaults to 'False'. If Substitution is given, it will be resolved at runtime.
```

Comment:
@kosuke55 if I may suggest, 

```py
:param: respawn if 'True', relaunch the process that abnormally died. 
   Either a boolean of a Substitution to be resolved at runtime. Defaults to 'False'. 
```

Reply:
@hidmic I updated the description following your suggestion. https://github.com/ros2/launch/pull/569/commits/afccdb12e4ec038eed3a3c4a2578703d74c1f41e

Reply:
Argh! I had a typo.

Reply:
Argh! I had a typo.

Reply:
Fixed in https://github.com/ros2/launch/pull/569/commits/80bbce6a56b0a79695f3a7637fe81742b6965239